packages/gallery/src/components/item/media/GalleryUI.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useGalleryUI } from '../../../context/GalleryContext';
+const galleryUiComponents = {
+  videoPlayButton: React.lazy(
+    () => import(/* webpackChunkName: "defaultPlayButton" */ './playButton')
+  ),
+  rotateArrow: React.lazy(
+    () => import(/* webpackChunkName: "defaultRotateArrow" */ './rotateArrow')
+  ),
+};
+interface GalleryUIProps {
+  size: number;
+  type: 'videoPlayButton' | 'rotateArrow';
+}
+
+export const GalleryUI = ({ type, size }: GalleryUIProps): JSX.Element => {
+  let Component;
+
+  const galleryUI = useGalleryUI();
+  if (typeof galleryUI?.[type] === 'function') {
```

Comment:
why would galleryUI.type would be a function

Reply:
galleryUI is an object of custom components